documentation/api/processes-get.md

```diff
@@ -0,0 +1,69 @@
+# Processes - Get
+
+Gets detailed information about a specified process.
```

Comment:
We may want to leave a note/link here for reverse server mode, since /processes will only show the list that's connected to the specific pipe.

Reply:
Yeah, I think I'll want to link to another doc that describes the connection modes. I'll do that in a subsequent change when that doc is available :-)